lib/taskana-data/src/test/java/pro/taskana/sampledata/SampleDataGeneratorTest.java

```diff
@@ -0,0 +1,26 @@
+package pro.taskana.sampledata;
+
+import java.util.stream.Stream;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+/**
+ * Test SampleDataGenerator.
+ */
+class SampleDataGeneratorTest {
+
+    @Test
+    void getScriptsNotEmpty() {
+        Stream<String> scripts = SampleDataGenerator.getDefaultScripts();
+        Assertions.assertNotNull(scripts);
+        Assertions.assertTrue(scripts.count() > 0);
+    }
+
+    @Test
+    void getScriptsFileExists() {
+        Stream<String> scripts = SampleDataGenerator.getDefaultScripts();
+        scripts.forEach(s -> Assertions.assertNotNull(SampleDataGenerator.getScriptBufferedStream(s)));
```

Comment:
SampleDataGenerator.getDefaultScripts()
.map(SampleDataGenerator::getScriptBufferedStream)
.forEach(Assertions::assertNotNull))

Reply:
what do you think of that? I <3 method references :) makes it look so "clean"

Reply:
done